Dockerfile

```diff
@@ -40,7 +40,7 @@ WORKDIR /home/epoch/node
 # Erl handle SIGQUIT instead of the default SIGINT
 STOPSIGNAL SIGQUIT
 
-EXPOSE 3013 3015 3113 3114
+EXPOSE 3013 3014 3015 3113 3114
```

Comment:
Could you please integrate [doc on how to join testnet using Docker and using state channels](https://github.com/aeternity/epoch/blob/061e7976aa061b30ddc9afdfc85defe591125b87/docs/docker.md#configuration)?